packages/testlab/src/request.ts

```diff
@@ -22,10 +22,15 @@ export function httpGetAsync(urlString: string): Promise<IncomingMessage> {
  * Async wrapper for making HTTPS GET requests
  * @param urlString
  */
-export function httpsGetAsync(urlString: string): Promise<IncomingMessage> {
-  const agent = new https.Agent({
-    rejectUnauthorized: false,
-  });
+export function httpsGetAsync(
+  urlString: string,
+  agent?: https.Agent,
+): Promise<IncomingMessage> {
+  agent =
+    agent ||
```

Comment:
I find the formatting of this code block rather weird. Have you considered the following alternative?

```ts
const defaultAgent = new https.Agent({
 rejectUnauthorized: false,
});

export function httpsGetAsync(
   urlString: string,
   agent: https.Agent = defaultAgent,
): Promise<IncomingMessage> {
  // agent is always set now
}
```